packages/epics/src/agreements/plugins/payment-schedule/payment-schedule.tsx

```
@@ -0,0 +1,215 @@
+    defaultValues: {
+      option: 'Immediately',
+      futureDate: undefined,
+      milestones: [],
+    },
+    mode: 'onChange',
+  });
+
+  return { methods, isInsideProvider };
+}
+
+export function PaymentSchedule({}: PaymentScheduleProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)